src/Compatibility/Core/src/Compatibility.csproj

```diff
@@ -52,6 +52,16 @@
   <ItemGroup Condition="$(TargetFramework.Contains('-windows')) == true ">
   </ItemGroup>
 
+  <PropertyGroup>
+    <InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);Microsoft.Maui.Controls.Generated</InterceptorsPreviewNamespaces>
```

Comment:
So, if an interceptor is declared outside of a namespace listed here, it will get a C# compiler error.

It sounds like they will rename this to `$(InterceptorsNamespaces)` one day:
* https://github.com/dotnet/csharplang/blob/main/proposals/interceptors-issues-2024-01.md#interceptorsnamespaces-property

No problem here, just fyi for later.